demos/demo_style_transfer.py

```diff
@@ -141,8 +140,13 @@ def transfer(c,s,alpha=1):
 ret, frame0 = cap.read()
 frame0=np.asfortranarray(np.array(frame0)/255)
 frame_webcam=frame0
-frame_style=frame0*0
+frame_style = frame0*0
 
+resize = False
+if frame_style.shape != (720, 1080, 3):
```

Comment:
This is  nice but I would like to put this condition (and the size of the reshape) as a parameter.

Could you use a system variable like I did with CAMERA that give the webcam number ? than you could call the scipt with :
`RESHAPE=1080x720 python demo_style_transfer.py`

and not reshape if no sizes are provided?

I tried it on my HD webcam and it gave weird ratios so I don't think it should be fixed

Reply:
I agree, some form of parameter would add flexibility. But in this case does it eve make sense to reshape to something else that 1080x720 if the network can only work with that resolution?

Reply:
The original script works on my HD webcam (and the low resolution one) with no need for reshape. The reshape parameter is definitely interesting because HD is very slow when no GPU is available though.

Do we really need the out to be exactly the same resolution as the input? it is shown in the different windows no?

It seems to me that it is hard to make OpenCV script work on different hardware ;).

Reply:
that's great thank you, yes you understood me very well.

I was thinking that maybe when no RESIZE shape is given we should not do the resize (which you do in the code if the native resolution of the webcam is not the default values  720, 1080).